src/Listeners/AddUserBioAttribute.php

```diff
@@ -23,9 +23,15 @@ class AddUserBioAttribute
      */
     protected $settings;
 
-    public function __construct(SettingsRepositoryInterface $settings)
+    /**
+     * @var Formatter
+     */
+    protected $formatter;
+
+    public function __construct(SettingsRepositoryInterface $settings, Formatter $formatter)
```

Comment:
To refer to the comment in the PR description, yes I think this approach is perfectly fine since we can't store anything on the user model class.

Reply:
We may need to create/store our own version of the formatter. Not sure if this is possible, but I assume we don't want people putting images/embeds in their bio? 🤷‍♂️